src/adapter/pg/read/target.go

```diff
@@ -0,0 +1,50 @@
+package read
+
+import (
+	"database/sql"
+	"github.com/liampm/static-code-analysis-tool/domain"
+	"github.com/satori/go.uuid"
+)
+
+type PostgresTargetReadRepo struct {
+	db *sql.DB
+}
+
+func TargetRepo(db *sql.DB) *PostgresTargetReadRepo {
+	return &PostgresTargetReadRepo{db: db}
+}
+
+func (repo *PostgresTargetReadRepo) Find(id uuid.UUID) (domain.Target, error) {
+	target := domain.Target{}
+
+	row := repo.db.QueryRow("SELECT * FROM target WHERE id = $1", id)
```

Comment:
I know I haven't done it for mine but you could just select the columns you need

Reply:
Actually, shouldn't this be selecting more? Otherwise it will just be returning a target with an ID and an empty project Id and name